File: app/src/main/java/com/church/punithaanthoniyar/utils/Commons.java
package com.church.punithaanthoniyar.utils;

import android.util.Log;

import com.church.punithaanthoniyar.BuildConfig;
import com.church.punithaanthoniyar.base.ApplicationConfigs;

public class Commons {
    public static final int DATE_TIME = 5;
    public static final int DATE = 6;

    public static void print( String message) {
        if (BuildConfig.DEBUG)
            Log.d(ApplicationConfigs.LOG, message);
    }

    public static void printException( String message, Throwable e) {
        Log.e(ApplicationConfigs.LOG, message, e);
    }

    public static void printException(Throwable e) {
        Log.e(ApplicationConfigs.LOG, "~", e);
    }

    public static void printException(String message) {
        Log.e(ApplicationConfigs.LOG, message);
    }

    public static void printInformation(String message) {
        if (BuildConfig.DEBUG)
            Log.i(ApplicationConfigs.LOG, message);
    }
}

File: app/src/main/java/com/church/punithaanthoniyar/utils/Commons.kt
package com.church.punithaanthoniyar.utils

import android.util.Log
import com.church.punithaanthoniyar.BuildConfig
import tbm.church.AppConstants

object Commons {
    const val DATE_TIME = 5
    const val DATE = 6
    fun print(message: String?) {}
    fun printException(message: String?, e: Throwable?) {
        Log.e(AppConstants.LOG, message, e)
    }

    fun printException(e: Throwable?) {
        Log.e(AppConstants.LOG, "~", e)
    }

    @JvmStatic
    fun printException(message: String?) {
        Log.e(AppConstants.LOG, message.toString())
    }

    fun printInformation(message: String?) {
        if (BuildConfig.DEBUG) Log.i(AppConstants.LOG, message.toString())
    }
}